### ISSUE TITLE
Global datapack doesn't seem to be working for me

### ISSUE THREAD

**Issue by gusbaruc** [Stereowalker/Reforged#45]
I made some changes to the mod including new modded weapons and tools and also removed the shields but the changes are not applied for some reason. I only managed to make it work when I replaced the files directly inside the .jar, opening it with an unzipper

Forge version: 43.2.11
Mod version: 5.0.3
Global Packs version: 1.14.2
Minecraft version: 1.19.2

**Comment by Stereowalker** [comment]
May I see your datapack?

**Comment by gusbaruc** [comment]
Sure! [Datapack](https://drive.google.com/file/d/1PE0MHvyk9SceLLmESaXWXIeFf6qHeqwi/view?usp=sharing)

I'm using the Global Packs mod to manage my modpack's datapacks.
When I test it does not reflect the changes. Neither the inclusion of items with the forge:paxels tag nor the removal of shields. Applying the changes to the .jar file directly works

**Comment by Stereowalker** [comment]
Paxels tag? Such tag does not exist in this mod

**Comment by gusbaruc** [comment]
From what I've seen the mod accepts any forge tag, including custom ones, so I created this tag. And it works! But like I said, just editing the file directly. I tried not using the tag and putting the item I wanted to reforge directly in the json file, but that didn't work either. I don't know if the problem is the datapack, to be honest. And even if that was the problem, at least the shields shouldn't get reforged, but even removing the tag from them, they still get it. Which reinforces the idea that for some reason the mod is not able to read the datapack.

**Comment by Stereowalker** [comment]
Makes sense. I just took a look at your datapack, and everything seems valid. Does the datapack show up in your world when you use the /datapack command?

**Comment by gusbaruc** [comment]
yes, it shows. This datapack has several other changes, and they are working. It could be something to do with the Global Pack as well, or perhaps packet prioritization. I don't know... Well, it must be something very specific. As a palliative solution I will continue to change the file directly.

**Comment by gusbaruc** [comment]
A little off topic, I wanted to leave a suggestion for adding an reforging table. That's what's missing for this mod to be perfect!

**Comment by Stereowalker** [comment]
Can I see your global packs config file? It might have what I need to crack this case.
The mod already has reforging. I explicitly didn't want to make it easy to do, so in order to do it you'll need the respective hammers for the item you want to reforge

**Comment by Stereowalker** [comment]
They are gotten from villager trading.
Weaponsmiths hammer reforges all weapons
Toolsmiths reforges all tools and 
Armorers hammer reforges armor. They're all gotten from their respective villagers at the highest level, and to use them just place then in the anvil with the item you want to reforge

**Comment by gusbaruc** [comment]
Sure!
```
#This should under no circumstances be set to true in modpacks
#If set to true, the mod creates a new global folder, that is global for your user on your computer.
#The created folders will be:
# <user_home>/.minecraft_global_packs/required_datapacks
# <user_home>/.minecraft_global_packs/optional_datapacks
# <user_home>/.minecraft_global_packs/required_resourcepacks
# <user_home>/.minecraft_global_packs/optional_resourcepacks
enable_system_global_packs = false

#This config allows you to add or remove folders or single files as Data-/Resourcepacks
#For that simply specify the file path to the folder/file relative to your .minecraft or instance folder
#With the current config you can place a Datapack into '.minecraft/global_packs/required_data' to force enable a datapack.
[resourcepacks]
required = [
    #"other_folder/random_pack.zip" # You can also point at specific files!
    "global_packs/required_resources/"
]

[datapacks]
required = [
    "resourcepacks/",
    "global_packs/required_data/"
]

optional = [
    "global_packs/optional_data/"
]
```

**Comment by gusbaruc** [comment]
Ahhh, cool! Now it makes sense. I saw these items for trading but didn't know what they were for. They have no description and do not appear in JEI.

I did some tests here, does it have a reforge limit? Apparently you can't reforge the same weapon multiple times. And if you allow me to make a new suggestion now that I understand how reforging works, I liked that it is a little more difficult to acquire, but I think the hammer should have greater durability. It is extremely expensive, since you have to increase the villager's level, then spend a large amount of emeralds and still spend experience on the anvil without even being sure that a good reforging will come. 

I imagine that all this work that the person will have to buy the hammer, ends up being much more viable for him to make a new weapon or armor.

**Comment by Stereowalker** [comment]
I can increase the durability a bit, but making reforging an inferior option to just crafting a new item was the goal. Although, considering I've  added pools, increasing durability might be a good idea. There is one final thing you can try. I'm going to do some testing on my own to figure out whats wrong, but do your see your datapack when you create a new world? It should show up in the list of datapacks

**Comment by gusbaruc** [comment]
Yes, it does. In this case, Tiered is inside that Painful & Spooky General. This datapack has changes for several mods, and all of them are working.
![image](https://github.com/Stereowalker/tiered/assets/57414280/5ad81ad0-6fe0-4325-8da6-38ced569a41a)
